### ISSUE TITLE
Docker container startup fail - JRE error

### ISSUE THREAD

**Issue by schlos** [metabase/metabase#43571]
### Describe the bug

Running [metabase/metabase:latest](app://dd/dashboard/images/local/sha256:a7da769be325f9e876e6ac3f05025e57e62c0ac432225f635579766d51373586?tag=latest&name=metabase%2Fmetabase) using Docker Desktop (upgraded to latest), docker fails with JRE error.

### To Reproduce

1. Go to Docker Desktop
2. Click on Metabase container
3. Press PLAY button (Run)

### Expected behavior

It should be started as per usual on port localhost:3000.

### Logs

2024-06-04 14:31:51 Warning: environ value jdk-11.0.23+9 for key :java-version has been overwritten with 11.0.23
2024-06-04 14:31:54 2024-06-04 04:31:54,504 INFO metabase.util :: Maximum memory available to JVM: 1.9 GB
2024-06-04 14:31:55 2024-06-04 04:31:55,920 WARN db.env :: WARNING: Using Metabase with an H2 application database is not recommended for production deployments. For production deployments, we highly recommend using Postgres, MySQL, or MariaDB instead. If you decide to continue to use H2, please be sure to back up the database file regularly. For more information, see https://metabase.com/docs/latest/operations-guide/migrating-from-h2.html
2024-06-04 14:31:56 2024-06-04 04:31:56,113 INFO util.encryption :: Saved credentials encryption is DISABLED for this Metabase instance. 🔓 
2024-06-04 14:31:56  For more information, see https://metabase.com/docs/latest/operations-guide/encrypting-database-details-at-rest.html
2024-06-04 14:31:59 [thread 49 also had an error]
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # A fatal error has been detected by the Java Runtime Environment:
2024-06-04 14:31:59 #
2024-06-04 14:31:59 #  SIGSEGV (0xb) at pc=0x00007ffffebd9f3d, pid=1, tid=52
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
2024-06-04 14:31:59 # Java VM: OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (11.0.23+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
2024-06-04 14:31:59 # Problematic frame:
2024-06-04 14:31:59 # V  [libjvm.so+0x78cf3d]  G1ParEvacuateFollowersClosure::do_void()+0x1fd
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # Can not save log file, dump to screen..
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # A fatal error has been detected by the Java Runtime Environment:
2024-06-04 14:31:59 #
2024-06-04 14:31:59 #  SIGSEGV (0xb) at pc=0x00007ffffebd9f3d, pid=1, tid=52
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
2024-06-04 14:31:59 # Java VM: OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (11.0.23+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
2024-06-04 14:31:59 # Problematic frame:
2024-06-04 14:31:59 # V  [libjvm.so+0x78cf3d]  G1ParEvacuateFollowersClosure::do_void()+0x1fd
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-06-04 14:31:59 #
2024-06-04 14:31:59 # If you would like to submit a bug report, please visit:
2024-06-04 14:31:59 #   https://github.com/adoptium/adoptium-support/issues
2024-06-04 14:31:59 #
2024-06-04 14:31:59 
2024-06-04 14:31:59 ---------------  S U M M A R Y ------------
2024-06-04 14:31:59 
2024-06-04 14:31:59 Command Line: -XX:+IgnoreUnrecognizedVMOptions -Dfile.encoding=UTF-8 -Dlogfile.path=target/log -XX:+CrashOnOutOfMemoryError /app/metabase.jar
2024-06-04 14:31:59 
2024-06-04 14:31:59 Host: x86_64, 8 cores, 7G, Alpine Linux v3.19
2024-06-04 14:31:59 Time: Tue Jun  4 04:31:59 2024 UTC elapsed time: 11.231443 seconds (0d 0h 0m 11s)
2024-06-04 14:31:59 
2024-06-04 14:31:59 ---------------  T H R E A D  ---------------
2024-06-04 14:31:59 
2024-06-04 14:31:59 Current thread (0x00007fffdc504000):  GCTaskThread "GC Thread#4" [stack: 0x00007fffdaa19000,0x00007fffdab19aa8] [id=52]
2024-06-04 14:31:59 
2024-06-04 14:31:59 Stack: [0x00007fffdaa19000,0x00007fffdab19aa8],  sp=0x00007fffdab19720,  free space=1025k
2024-06-04 14:31:59 Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
2024-06-04 14:31:59 V  [libjvm.so+0x78cf3d]  G1ParEvacuateFollowersClosure::do_void()+0x1fd
2024-06-04 14:31:59 V  [libjvm.so+0x794c66]  G1ParTask::work(unsigned int)+0x166
2024-06-04 14:31:59 V  [libjvm.so+0xed7abd]  GangWorker::loop()+0x4d
2024-06-04 14:31:59 V  [libjvm.so+0xe3ddfc]  Thread::call_run()+0x13c
2024-06-04 14:31:59 V  [libjvm.so+0xc28036]  thread_native_entry(Thread*)+0x136
2024-06-04 14:31:59 
2024-06-04 14:31:59 
2024-06-04 14:31:59 siginfo: si_signo: 11 (SIGSEGV), si_code: 1 (SEGV_MAPERR), si_addr: 0x0000000000000000
2024-06-04 14:31:59 
2024-06-04 14:31:59 Register to memory mapping:
2024-06-04 14:31:59 
2024-06-04 14:31:59 RAX=0x00000c3200000b01 is an unknown value
2024-06-04 14:31:59 RBX=0x0 is NULL
2024-06-04 14:31:59 RCX=0x0000000000000b56 is an unknown value
2024-06-04 14:31:59 RDX=0x00000c3200000b57 is an unknown value
.....
2024-06-04 14:31:59 VM Arguments:
2024-06-04 14:31:59 jvm_args: -XX:+IgnoreUnrecognizedVMOptions -Dfile.encoding=UTF-8 -Dlogfile.path=target/log -XX:+CrashOnOutOfMemoryError 
2024-06-04 14:31:59 java_command: /app/metabase.jar
2024-06-04 14:31:59 java_class_path (initial): /app/metabase.jar
2024-06-04 14:31:59 Launcher Type: SUN_STANDARD
2024-06-04 14:31:59 
2024-06-04 14:31:59 [Global flags]
2024-06-04 14:31:59      intx CICompilerCount                          = 4                                         {product} {ergonomic}
2024-06-04 14:31:59      uint ConcGCThreads                            = 2                                         {product} {ergonomic}
2024-06-04 14:31:59      bool CrashOnOutOfMemoryError                  = true                                      {product} {command line}
2024-06-04 14:31:59      uint G1ConcRefinementThreads                  = 8                                         {product} {ergonomic}
2024-06-04 14:31:59    size_t G1HeapRegionSize                         = 1048576                                   {product} {ergonomic}
2024-06-04 14:31:59     uintx GCDrainStackTargetSize                   = 64                                        {product} {ergonomic}
2024-06-04 14:31:59      bool IgnoreUnrecognizedVMOptions              = true                                      {product} {command line}
2024-06-04 14:31:59    size_t InitialHeapSize                          = 130023424                                 {product} {ergonomic}
2024-06-04 14:31:59    size_t MarkStackSize                            = 4194304                                   {product} {ergonomic}
2024-06-04 14:31:59    size_t MaxHeapSize                              = 2057306112                                {product} {ergonomic}
2024-06-04 14:31:59    size_t MaxNewSize                               = 1234173952                                {product} {ergonomic}
2024-06-04 14:31:59    size_t MinHeapDeltaBytes                        = 1048576                                   {product} {ergonomic}
2024-06-04 14:31:59     uintx NonNMethodCodeHeapSize                   = 5836300                                {pd product} {ergonomic}
2024-06-04 14:31:59     uintx NonProfiledCodeHeapSize                  = 122910970                              {pd product} {ergonomic}
2024-06-04 14:31:59     uintx ProfiledCodeHeapSize                     = 122910970                              {pd product} {ergonomic}
2024-06-04 14:31:59     uintx ReservedCodeCacheSize                    = 251658240                              {pd product} {ergonomic}
2024-06-04 14:31:59      bool SegmentedCodeCache                       = true                                      {product} {ergonomic}
2024-06-04 14:31:59      bool UseCompressedClassPointers               = true                                 {lp64_product} {ergonomic}
2024-06-04 14:31:59      bool UseCompressedOops                        = true                                 {lp64_product} {ergonomic}
2024-06-04 14:31:59      bool UseG1GC                                  = true                                      {product} {ergonomic}
2024-06-04 14:31:59 
2024-06-04 14:31:59 Logging:
2024-06-04 14:31:59 Log output configuration:
2024-06-04 14:31:59  #0: stdout all=warning uptime,level,tags
2024-06-04 14:31:59  #1: stderr all=off uptime,level,tags
2024-06-04 14:31:59 
2024-06-04 14:31:59 Environment Variables:
2024-06-04 14:31:59 JAVA_HOME=/opt/java/openjdk
2024-06-04 14:31:59 PATH=/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
2024-06-04 14:31:59 FC_LANG=en-US
2024-06-04 14:31:59 LD_LIBRARY_PATH=/opt/java/openjdk/lib/server:/opt/java/openjdk/lib:/opt/java/openjdk/../lib
2024-06-04 14:31:59 SHELL=/bin/sh
2024-06-04 14:31:59 LANG=en_US.UTF-8
2024-06-04 14:31:59 LC_ALL=en_US.UTF-8
2024-06-04 14:31:59 LC_CTYPE=en_US.UTF-8
2024-06-04 14:31:59 
2024-06-04 14:31:59 Signal Handlers:
2024-06-04 14:31:59 SIGSEGV: [libjvm.so+0xeab920], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGBUS: [libjvm.so+0xeab920], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGFPE: [libjvm.so+0xeab920], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGPIPE: [libjvm.so+0xc1e340], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGXFSZ: [libjvm.so+0xc1e340], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGILL: [libjvm.so+0xeab920], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGUSR2: [libjvm.so+0xc1e1e0], sa_mask[0]=00000000000000000000000000000000, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGHUP: [libjvm.so+0xc1e990], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGINT: [libjvm.so+0xc1e990], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGTERM: [libjvm.so+0xc1e990], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 SIGQUIT: [libjvm.so+0xc1e990], sa_mask[0]=11111111111111111111111111111110, sa_flags=SA_RESTART|SA_SIGINFO
2024-06-04 14:31:59 
2024-06-04 14:31:59 
2024-06-04 14:31:59 ---------------  S Y S T E M  ---------------
2024-06-04 14:31:59 
2024-06-04 14:31:59 OS:NAME="Alpine Linux"
2024-06-04 14:31:59 ID=alpine
2024-06-04 14:31:59 VERSION_ID=3.19.1
2024-06-04 14:31:59 PRETTY_NAME="Alpine Linux v3.19"
2024-06-04 14:31:59 HOME_URL="https://alpinelinux.org/"
2024-06-04 14:31:59 BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
2024-06-04 14:31:59 uname:Linux 6.6.26-linuxkit #1 SMP Sat Apr 27 04:13:19 UTC 2024 x86_64
2024-06-04 14:31:59 OS uptime: 0 days 0:00 hours
2024-06-04 14:31:59 libc:musl - unknown musl - unknown 
2024-06-04 14:31:59 rlimit (soft/hard): STACK 8192k/infinity , CORE 0k/infinity , NPROC infinity/infinity , NOFILE 1048576/1048576 , AS infinity/infinity , CPU infinity/infinity , DATA infinity/infinity , FSIZE infinity/infinity , MEMLOCK infinity/infinity
2024-06-04 14:31:59 load average:0.46 0.10 0.03
2024-06-04 14:31:59 
2024-06-04 14:31:59 /proc/meminfo:
2024-06-04 14:31:59 MemTotal:        8029248 kB
2024-06-04 14:31:59 MemFree:         6743444 kB
2024-06-04 14:31:59 MemAvailable:    6942680 kB
2024-06-04 14:31:59 Buffers:           10392 kB
2024-06-04 14:31:59 Cached:           330920 kB
2024-06-04 14:31:59 SwapCached:            0 kB
2024-06-04 14:31:59 Active:          1054564 kB
2024-06-04 14:31:59 Inactive:         114504 kB
2024-06-04 14:31:59 Active(anon):     828452 kB
2024-06-04 14:31:59 Inactive(anon):        0 kB
2024-06-04 14:31:59 Active(file):     226112 kB
2024-06-04 14:31:59 Inactive(file):   114504 kB
2024-06-04 14:31:59 Unevictable:           0 kB
2024-06-04 14:31:59 Mlocked:               0 kB
2024-06-04 14:31:59 SwapTotal:       1048572 kB
2024-06-04 14:31:59 SwapFree:        1048572 kB
2024-06-04 14:31:59 Zswap:                 0 kB
2024-06-04 14:31:59 Zswapped:              0 kB
2024-06-04 14:31:59 Dirty:               388 kB
2024-06-04 14:31:59 Writeback:             0 kB
2024-06-04 14:31:59 AnonPages:        826944 kB
2024-06-04 14:31:59 Mapped:           135828 kB
2024-06-04 14:31:59 Shmem:               640 kB
2024-06-04 14:31:59 KReclaimable:      10304 kB
2024-06-04 14:31:59 Slab:              46100 kB
2024-06-04 14:31:59 SReclaimable:      10304 kB
2024-06-04 14:31:59 SUnreclaim:        35796 kB
2024-06-04 14:31:59 KernelStack:        4592 kB
2024-06-04 14:31:59 PageTables:         3468 kB
2024-06-04 14:31:59 SecPageTables:         0 kB
2024-06-04 14:31:59 NFS_Unstable:          0 kB
2024-06-04 14:31:59 Bounce:                0 kB
2024-06-04 14:31:59 WritebackTmp:          0 kB
2024-06-04 14:31:59 CommitLimit:     5063196 kB
2024-06-04 14:31:59 Committed_AS:    1755168 kB
2024-06-04 14:31:59 VmallocTotal:   133141626880 kB
2024-06-04 14:31:59 VmallocUsed:        2448 kB
2024-06-04 14:31:59 VmallocChunk:          0 kB
2024-06-04 14:31:59 Percpu:             2208 kB
2024-06-04 14:31:59 AnonHugePages:    475136 kB
2024-06-04 14:31:59 ShmemHugePages:        0 kB
2024-06-04 14:31:59 ShmemPmdMapped:        0 kB
2024-06-04 14:31:59 FileHugePages:         0 kB
2024-06-04 14:31:59 FilePmdMapped:         0 kB
2024-06-04 14:31:59 HugePages_Total:       0
2024-06-04 14:31:59 HugePages_Free:        0
2024-06-04 14:31:59 HugePages_Rsvd:        0
2024-06-04 14:31:59 HugePages_Surp:        0
2024-06-04 14:31:59 Hugepagesize:       2048 kB
2024-06-04 14:31:59 Hugetlb:               0 kB
2024-06-04 14:31:59 
2024-06-04 14:31:59 /sys/kernel/mm/transparent_hugepage/enabled:
2024-06-04 14:31:59 [always] madvise never
2024-06-04 14:31:59 /sys/kernel/mm/transparent_hugepage/defrag (defrag/compaction efforts parameter):
2024-06-04 14:31:59 always defer defer+madvise [madvise] never
2024-06-04 14:31:59 
2024-06-04 14:31:59 Process Memory:
2024-06-04 14:31:59 Virtual Size: 4100116K (peak: 4121480K)
2024-06-04 14:31:59 Resident Set Size: 663624K (peak: 676124K) (anon: 645320K, file: 18304K, shmem: 0K)
2024-06-04 14:31:59 Swapped out: 0K
2024-06-04 14:31:59 
2024-06-04 14:31:59 /proc/sys/kernel/threads-max (system-wide limit on the number of threads):
2024-06-04 14:31:59 62685
2024-06-04 14:31:59 /proc/sys/vm/max_map_count (maximum number of memory map areas a process may have):
2024-06-04 14:31:59 262144
2024-06-04 14:31:59 /proc/sys/kernel/pid_max (system-wide limit on number of process identifiers):
2024-06-04 14:31:59 99999
2024-06-04 14:31:59
...
2024-06-04 14:31:59 
2024-06-04 14:31:59 Memory: 4k page, physical 8029248k(6743192k free), swap 1048572k(1048572k free)
2024-06-04 14:31:59 
2024-06-04 14:31:59 vm_info: OpenJDK 64-Bit Server VM (11.0.23+9) for linux-amd64-musl JRE (11.0.23+9), built on Apr 17 2024 05:45:29 by "" with gcc 10.3.1 20211027
2024-06-04 14:31:59 
2024-06-04 14:31:59 END.

### Information about your Metabase installation

```JSON
- OS: MacOS Sonoma 14.5
- Apple M3 chip
- Docker Desktop v4.30.0
```

### Severity

Blocker for docker image user

**Comment by schlos** [comment]
Tried also other docker container runtime to rule out issue with Docker Desktop - tried on OrbStack (latest version, 1.6.1), error logged there:

```
Warning: environ value jdk-11.0.23+9 for key :java-version has been overwritten with 11.0.23
2024-06-04 04:55:38,018 INFO metabase.util :: Maximum memory available to JVM: 1.3 GB
2024-06-04 04:55:39,380 WARN db.env :: WARNING: Using Metabase with an H2 application database is not recommended for production deployments. For production deployments, we highly recommend using Postgres, MySQL, or MariaDB instead. If you decide to continue to use H2, please be sure to back up the database file regularly. For more information, see https://metabase.com/docs/latest/operations-guide/migrating-from-h2.html
2024-06-04 04:55:39,536 INFO util.encryption :: Saved credentials encryption is DISABLED for this Metabase instance. 🔓 
 For more information, see https://metabase.com/docs/latest/operations-guide/encrypting-database-details-at-rest.html
2024-06-04 04:55:43,705 INFO driver.impl :: Registered abstract driver :sql  🚚
2024-06-04 04:55:43,714 INFO driver.impl :: Registered abstract driver :sql-jdbc (parents: [:sql]) 🚚
2024-06-04 04:55:43,722 INFO metabase.util :: Load driver :sql-jdbc took 49.8 ms
2024-06-04 04:55:43,724 INFO driver.impl :: Registered driver :h2 (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:43,877 INFO driver.impl :: Registered driver :mysql (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:43,905 INFO driver.impl :: Registered driver :postgres (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:45,214 INFO metabase.core :: 
Metabase v0.49.13 (de28e83) 

Copyright © 2024 Metabase, Inc. 

Metabase Enterprise Edition extensions are NOT PRESENT.
2024-06-04 04:55:45,224 INFO metabase.core :: Starting Metabase in STANDALONE mode
2024-06-04 04:55:45,276 INFO metabase.server :: Launching Embedded Jetty Webserver with config:
 {:port 3000, :host "0.0.0.0"}

2024-06-04 04:55:45,332 INFO metabase.core :: Starting Metabase version v0.49.13 (de28e83) ...
2024-06-04 04:55:45,336 INFO metabase.core :: System info:
 {"file.encoding" "UTF-8",
 "java.runtime.name" "OpenJDK Runtime Environment",
 "java.runtime.version" "11.0.23+9",
 "java.vendor" "Eclipse Adoptium",
 "java.vendor.url" "https://adoptium.net/",
 "java.version" "11.0.23",
 "java.vm.name" "OpenJDK 64-Bit Server VM",
 "java.vm.version" "11.0.23+9",
 "os.name" "Linux",
 "os.version" "6.7.12-orbstack-00202-g57474688ffbd",
 "user.language" "en",
 "user.timezone" "GMT"}

2024-06-04 04:55:45,340 INFO metabase.plugins :: Loading plugins in /plugins...
2024-06-04 04:55:45,447 INFO util.files :: Extract file /modules/sparksql.metabase-driver.jar -> /plugins/sparksql.metabase-driver.jar
2024-06-04 04:55:45,510 INFO util.files :: Extract file /modules/sqlserver.metabase-driver.jar -> /plugins/sqlserver.metabase-driver.jar
2024-06-04 04:55:45,519 INFO util.files :: Extract file /modules/snowflake.metabase-driver.jar -> /plugins/snowflake.metabase-driver.jar
2024-06-04 04:55:45,781 INFO util.files :: Extract file /modules/mongo.metabase-driver.jar -> /plugins/mongo.metabase-driver.jar
2024-06-04 04:55:45,794 INFO util.files :: Extract file /modules/oracle.metabase-driver.jar -> /plugins/oracle.metabase-driver.jar
2024-06-04 04:55:45,795 INFO util.files :: Extract file /modules/googleanalytics.metabase-driver.jar -> /plugins/googleanalytics.metabase-driver.jar
2024-06-04 04:55:45,804 INFO util.files :: Extract file /modules/athena.metabase-driver.jar -> /plugins/athena.metabase-driver.jar
2024-06-04 04:55:45,860 INFO util.files :: Extract file /modules/redshift.metabase-driver.jar -> /plugins/redshift.metabase-driver.jar
2024-06-04 04:55:45,864 INFO util.files :: Extract file /modules/presto-jdbc.metabase-driver.jar -> /plugins/presto-jdbc.metabase-driver.jar
2024-06-04 04:55:45,900 INFO util.files :: Extract file /modules/bigquery-cloud-sdk.metabase-driver.jar -> /plugins/bigquery-cloud-sdk.metabase-driver.jar
2024-06-04 04:55:46,009 INFO util.files :: Extract file /modules/druid.metabase-driver.jar -> /plugins/druid.metabase-driver.jar
2024-06-04 04:55:46,011 INFO util.files :: Extract file /modules/vertica.metabase-driver.jar -> /plugins/vertica.metabase-driver.jar
2024-06-04 04:55:46,012 INFO util.files :: Extract file /modules/sqlite.metabase-driver.jar -> /plugins/sqlite.metabase-driver.jar
2024-06-04 04:55:46,214 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :hive-like...
2024-06-04 04:55:46,217 INFO driver.impl :: Registered abstract driver :hive-like (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,220 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sparksql...
2024-06-04 04:55:46,221 INFO driver.impl :: Registered driver :sparksql (parents: [:hive-like]) 🚚
2024-06-04 04:55:46,231 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sqlserver...
2024-06-04 04:55:46,232 INFO driver.impl :: Registered driver :sqlserver (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,265 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :snowflake...
2024-06-04 04:55:46,266 INFO driver.impl :: Registered driver :snowflake (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,275 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :mongo...
2024-06-04 04:55:46,276 INFO driver.impl :: Registered driver :mongo  🚚
2024-06-04 04:55:46,296 INFO plugins.dependencies :: Metabase cannot initialize plugin Metabase Oracle Driver due to required dependencies. Metabase requires the Oracle JDBC driver in order to connect to Oracle databases, but we can't ship it as part of Metabase due to licensing restrictions. See https://metabase.com/docs/latest/administration-guide/databases/oracle.html for more details.

2024-06-04 04:55:46,297 INFO plugins.dependencies :: Metabase Oracle Driver dependency {:class oracle.jdbc.OracleDriver} satisfied? false
2024-06-04 04:55:46,298 INFO plugins.dependencies :: Plugins with unsatisfied deps: ["Metabase Oracle Driver"]
2024-06-04 04:55:46,312 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :googleanalytics...
2024-06-04 04:55:46,313 INFO driver.impl :: Registered driver :googleanalytics  🚚
2024-06-04 04:55:46,326 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :athena...
2024-06-04 04:55:46,327 INFO driver.impl :: Registered driver :athena (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,330 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :redshift...
2024-06-04 04:55:46,330 INFO driver.impl :: Registered driver :redshift (parents: [:postgres]) 🚚
2024-06-04 04:55:46,345 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :presto-jdbc...
2024-06-04 04:55:46,345 INFO driver.impl :: Registered driver :presto-jdbc (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,356 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :bigquery-cloud-sdk...
2024-06-04 04:55:46,357 INFO driver.impl :: Registered driver :bigquery-cloud-sdk (parents: [:sql]) 🚚
2024-06-04 04:55:46,363 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :druid...
2024-06-04 04:55:46,364 INFO driver.impl :: Registered driver :druid  🚚
2024-06-04 04:55:46,369 INFO plugins.dependencies :: Metabase cannot initialize plugin Metabase Vertica Driver due to required dependencies. Metabase requires the Vertica JDBC driver in order to connect to Vertica databases, but we can't ship it as part of Metabase due to licensing restrictions. See https://metabase.com/docs/latest/administration-guide/databases/vertica.html for more details.

2024-06-04 04:55:46,371 INFO plugins.dependencies :: Metabase Vertica Driver dependency {:class com.vertica.jdbc.Driver} satisfied? false
2024-06-04 04:55:46,372 INFO plugins.dependencies :: Plugins with unsatisfied deps: ["Metabase Vertica Driver" "Metabase Oracle Driver"]
2024-06-04 04:55:46,373 DEBUG plugins.lazy-loaded-driver :: Registering lazy loading driver :sqlite...
2024-06-04 04:55:46,374 INFO driver.impl :: Registered driver :sqlite (parents: [:sql-jdbc]) 🚚
2024-06-04 04:55:46,387 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
2024-06-04 04:55:46,389 INFO db.setup :: Verifying h2 Database Connection ...
2024-06-04 04:55:47,129 INFO db.setup :: Successfully verified H2 2.1.214 (2022-06-13) application database connection. ✅
2024-06-04 04:55:47,132 INFO db.setup :: Checking if a database downgrade is required...
2024-06-04 04:55:47,851 INFO db.setup :: Running Database Migrations...
2024-06-04 04:55:47,852 INFO db.setup :: Setting up Liquibase...
2024-06-04 04:55:48,131 INFO db.setup :: Liquibase is ready.
2024-06-04 04:55:48,134 INFO db.liquibase :: Checking if Database has unrun migrations...
2024-06-04 04:55:48,659 INFO db.liquibase :: Database has unrun migrations. Checking if migraton lock is taken...
2024-06-04 04:55:48,671 INFO db.liquibase :: No migration lock found.
2024-06-04 04:55:48,936 INFO db.liquibase :: Running 9 migrations ...
2024-06-04 04:55:49,428 INFO impl.StdSchedulerFactory :: Using default implementation for ThreadExecutor
2024-06-04 04:55:49,459 INFO core.SchedulerSignalerImpl :: Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2024-06-04 04:55:49,460 INFO core.QuartzScheduler :: Quartz Scheduler v.2.3.2 created.
2024-06-04 04:55:49,460 INFO jdbcjobstore.JobStoreTX :: Using db table-based data access locking (synchronization).
2024-06-04 04:55:49,462 INFO jdbcjobstore.JobStoreTX :: JobStoreTX initialized.
2024-06-04 04:55:49,465 INFO core.QuartzScheduler :: Scheduler meta-data: Quartz Scheduler (v2.3.2) 'MetabaseScheduler' with instanceId 'ab66a974b5581717476949439'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 10 threads.
  Using job-store 'org.quartz.impl.jdbcjobstore.JobStoreTX' - which supports persistence. and is clustered.

2024-06-04 04:55:49,465 INFO impl.StdSchedulerFactory :: Quartz scheduler 'MetabaseScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2024-06-04 04:55:49,465 INFO impl.StdSchedulerFactory :: Quartz scheduler version: 2.3.2
2024-06-04 04:55:49,510 INFO jdbcjobstore.JobStoreTX :: ClusterManager: detected 1 failed or restarted instances.
2024-06-04 04:55:49,510 INFO jdbcjobstore.JobStoreTX :: ClusterManager: Scanning for instance "60f0e50b3bfa1716200268272"'s failed in-progress jobs.
2024-06-04 04:55:49,531 INFO jdbcjobstore.JobStoreTX :: ClusterManager: ......Cleaned-up 1 other failed job(s).
2024-06-04 04:55:49,545 INFO core.QuartzScheduler :: Scheduler MetabaseScheduler_$_ab66a974b5581717476949439 started.
2024-06-04 04:55:49,592 INFO core.QuartzScheduler :: Scheduler MetabaseScheduler_$_ab66a974b5581717476949439 shutting down.
2024-06-04 04:55:49,592 INFO core.QuartzScheduler :: Scheduler MetabaseScheduler_$_ab66a974b5581717476949439 paused.
2024-06-04 04:55:49,602 INFO jdbcjobstore.JobStoreTX :: Handling the first 20 triggers that missed their scheduled fire-time.  More misfired triggers remain to be processed.
2024-06-04 04:55:49,615 ERROR jdbcjobstore.JobStoreTX :: MisfireHandler: Error handling misfires: Couldn't retrieve trigger: Block not found in id [1, -26, 2, -28, 2] [2.1.214/50]
org.quartz.JobPersistenceException: Couldn't retrieve trigger: Block not found in id [1, -26, 2, -28, 2] [2.1.214/50] [See nested exception: java.io.IOException: Block not found in id [1, -26, 2, -28, 2] [2.1.214/50]]
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1538)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:984)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3264)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:4012)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:4033)
Caused by: java.io.IOException: Block not found in id [1, -26, 2, -28, 2] [2.1.214/50]
	at org.h2.mvstore.StreamStore$Stream.read(StreamStore.java:514)
	at java.base/java.io.FilterInputStream.read(Unknown Source)
	at org.h2.mvstore.db.LobStorageMap$LobInputStream.read(LobStorageMap.java:358)
	at java.base/java.io.ObjectInputStream$PeekInputStream.read(Unknown Source)
	at java.base/java.io.ObjectInputStream$PeekInputStream.readFully(Unknown Source)
	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readShort(Unknown Source)
	at java.base/java.io.ObjectInputStream.readStreamHeader(Unknown Source)
	at java.base/java.io.ObjectInputStream.<init>(Unknown Source)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.getObjectFromBlob(StdJDBCDelegate.java:3199)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTrigger(StdJDBCDelegate.java:1780)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1536)
	... 4 more
Caused by: org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@4b737e1c failed at 3129518 (length -1), read 0, remaining 384 [2.1.214/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:470)
	at org.h2.mvstore.FileStore.readFully(FileStore.java:98)
	at org.h2.mvstore.Chunk.readBufferForPage(Chunk.java:422)
	at org.h2.mvstore.MVStore.readPage(MVStore.java:2569)
	at org.h2.mvstore.MVMap.readPage(MVMap.java:633)
	at org.h2.mvstore.Page$NonLeaf.getChildPage(Page.java:1125)
	at org.h2.mvstore.Page.get(Page.java:243)
	at org.h2.mvstore.MVMap.get(MVMap.java:436)
	at org.h2.mvstore.MVMap.get(MVMap.java:424)
	at org.h2.mvstore.StreamStore.getBlock(StreamStore.java:434)
	at org.h2.mvstore.StreamStore$Stream.nextBuffer(StreamStore.java:551)
	at org.h2.mvstore.StreamStore$Stream.read(StreamStore.java:508)
	... 14 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(Unknown Source)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(Unknown Source)
	at java.base/sun.nio.ch.FileChannelImpl.readInternal(Unknown Source)
	at java.base/sun.nio.ch.FileChannelImpl.read(Unknown Source)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:456)
	... 25 more
2024-06-04 04:55:49,622 INFO core.QuartzScheduler :: Scheduler MetabaseScheduler_$_ab66a974b5581717476949439 shutdown complete.
2024-06-04 04:55:49,808 ERROR liquibase.changelog :: ChangeSet migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat encountered an exception.

UPDATE SUMMARY
Run:                          9
Previously run:             262
Filtered out:                19
-------------------------------
Total change sets:          290


FILTERED CHANGE SETS SUMMARY
DBMS mismatch:               19

2024-06-04 04:55:49,873 ERROR metabase.core :: Metabase Initialization FAILED
liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat:
     Reason: clojure.lang.ExceptionInfo: General error: "General error: ""org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]"" [50000-214]"; SQL statement:
COMMIT [50000-214] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.h2.jdbc.JdbcConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
	at liquibase.command.CommandScope.execute(CommandScope.java:253)
	at liquibase.Liquibase.lambda$update$0(Liquibase.java:245)
	at liquibase.Scope.lambda$child$0(Scope.java:186)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:185)
	at liquibase.Scope.child(Scope.java:164)
	at liquibase.Liquibase.runInScope(Liquibase.java:1419)
	at liquibase.Liquibase.update(Liquibase.java:234)
	at liquibase.Liquibase.update(Liquibase.java:212)
	at liquibase.Liquibase.update(Liquibase.java:194)
	at metabase.db.liquibase$migrate_up_if_needed_BANG_.invokeStatic(liquibase.clj:305)
	at metabase.db.liquibase$migrate_up_if_needed_BANG_.invoke(liquibase.clj:287)
	at metabase.db.setup$migrate_BANG_$fn__51159.invoke(setup.clj:80)
	at metabase.db.liquibase$do_with_liquibase$f_STAR___48835.invoke(liquibase.clj:139)
	at metabase.db.liquibase$do_with_liquibase.invokeStatic(liquibase.clj:142)
	at metabase.db.liquibase$do_with_liquibase.invoke(liquibase.clj:130)
	at metabase.db.setup$migrate_BANG_.invokeStatic(setup.clj:75)
	at metabase.db.setup$migrate_BANG_.doInvoke(setup.clj:56)
	at clojure.lang.RestFn.invoke(RestFn.java:445)
	at metabase.db.setup$run_schema_migrations_BANG_.invokeStatic(setup.clj:147)
	at metabase.db.setup$run_schema_migrations_BANG_.invoke(setup.clj:141)
	at metabase.db.setup$setup_db_BANG_$fn__51187$fn__51188.invoke(setup.clj:165)
	at metabase.util.jvm$do_with_us_locale.invokeStatic(jvm.clj:239)
	at metabase.util.jvm$do_with_us_locale.invoke(jvm.clj:225)
	at metabase.db.setup$setup_db_BANG_$fn__51187.invoke(setup.clj:160)
	at metabase.db.setup$setup_db_BANG_.invokeStatic(setup.clj:159)
	at metabase.db.setup$setup_db_BANG_.invoke(setup.clj:153)
	at metabase.db$setup_db_BANG_$fn__51207.invoke(db.clj:69)
	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:64)
	at metabase.db$setup_db_BANG_.invoke(db.clj:55)
	at metabase.core$init_BANG__STAR_.invokeStatic(core.clj:116)
	at metabase.core$init_BANG__STAR_.invoke(core.clj:101)
	at metabase.core$init_BANG_.invokeStatic(core.clj:159)
	at metabase.core$init_BANG_.invoke(core.clj:154)
	at metabase.core$start_normally.invokeStatic(core.clj:171)
	at metabase.core$start_normally.invoke(core.clj:165)
	at metabase.core$entrypoint.invokeStatic(core.clj:204)
	at metabase.core$entrypoint.doInvoke(core.clj:198)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
	at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.bootstrap.main(Unknown Source)
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat:
     Reason: clojure.lang.ExceptionInfo: General error: "General error: ""org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]"" [50000-214]"; SQL statement:
COMMIT [50000-214] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.h2.jdbc.JdbcConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:151)
	at liquibase.command.core.AbstractUpdateCommandStep.lambda$run$0(AbstractUpdateCommandStep.java:110)
	at liquibase.Scope.lambda$child$0(Scope.java:186)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:185)
	at liquibase.Scope.child(Scope.java:164)
	at liquibase.command.core.AbstractUpdateCommandStep.run(AbstractUpdateCommandStep.java:108)
	at liquibase.command.core.UpdateCommandStep.run(UpdateCommandStep.java:105)
	at liquibase.command.CommandScope.execute(CommandScope.java:217)
	... 49 more
Caused by: liquibase.exception.MigrationFailedException: Migration failed for changeset migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat:
     Reason: clojure.lang.ExceptionInfo: General error: "General error: ""org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]"" [50000-214]"; SQL statement:
COMMIT [50000-214] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.h2.jdbc.JdbcConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:797)
	at liquibase.changelog.visitor.UpdateVisitor.executeAcceptedChange(UpdateVisitor.java:119)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:68)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$0(ChangeLogIterator.java:133)
	at liquibase.Scope.lambda$child$0(Scope.java:186)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:185)
	at liquibase.Scope.child(Scope.java:164)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:122)
	at liquibase.Scope.lambda$child$0(Scope.java:186)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:185)
	at liquibase.Scope.child(Scope.java:164)
	at liquibase.Scope.child(Scope.java:252)
	at liquibase.Scope.child(Scope.java:256)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:89)
	... 57 more
Caused by: clojure.lang.ExceptionInfo: General error: "General error: ""org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]"" [50000-214]"; SQL statement:
COMMIT [50000-214] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.h2.jdbc.JdbcConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.Database.throwLastBackgroundException(Database.java:1782)
	at org.h2.engine.SessionLocal.beforeCommitOrRollback(SessionLocal.java:754)
	at org.h2.engine.SessionLocal.commit(SessionLocal.java:668)
	at org.h2.command.dml.TransactionCommand.update(TransactionCommand.java:46)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcConnection.commit(JdbcConnection.java:447)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.commit(NewProxyConnection.java:981)
	at metabase.db.connection$do_transaction$thunk__32333.invoke(connection.clj:153)
	at metabase.db.connection$do_transaction.invokeStatic(connection.clj:165)
	at metabase.db.connection$do_transaction.invoke(connection.clj:146)
	at metabase.db.connection$do_with_transaction_primary_method_java_sql_Connection.invokeStatic(connection.clj:199)
	at metabase.db.connection$do_with_transaction_primary_method_java_sql_Connection.invoke(connection.clj:172)
	at clojure.lang.AFn.applyToHelper(AFn.java:165)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:457)
	at clojure.core$partial$fn__5908.invoke(core.clj:2643)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at methodical.impl.combo.threaded$fn__18243$fn__18244$fn__18247.invoke(threaded.clj:71)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213$fn__18217.invoke(threaded.clj:23)
	at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
	at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
	at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6887)
	at clojure.core$reduce.invoke(core.clj:6869)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213.invoke(threaded.clj:21)
	at clojure.core$comp$fn__5876.invoke(core.clj:2588)
	at methodical.impl.combo.threaded$combine_with_threader$fn__18223.invoke(threaded.clj:44)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at toucan2.connection$do_with_transaction_around_method_toucan2_connection_default.invokeStatic(connection.clj:249)
	at toucan2.connection$do_with_transaction_around_method_toucan2_connection_default.invoke(connection.clj:245)
	at clojure.lang.AFn.applyToHelper(AFn.java:165)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:457)
	at clojure.core$partial$fn__5908.invoke(core.clj:2643)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:58)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:195)
	at metabase.db.custom_migrations.DeleteScanFieldValuesTriggerForDBThatTurnItOff$with_connection_STAR___48618.invoke(custom_migrations.clj:1106)
	at toucan2.connection$bind_current_connectable_fn$fn__21153.invoke(connection.clj:104)
	at toucan2.connection$bind_current_connectable_fn$fn__21153.invoke(connection.clj:104)
	at toucan2.jdbc.connection$do_with_connection_primary_method_java_sql_Connection.invokeStatic(connection.clj:13)
	at toucan2.jdbc.connection$do_with_connection_primary_method_java_sql_Connection.invoke(connection.clj:11)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at clojure.core$partial$fn__5908.invoke(core.clj:2642)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at methodical.impl.combo.threaded$fn__18243$fn__18244$fn__18245.invoke(threaded.clj:70)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213$fn__18217.invoke(threaded.clj:23)
	at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
	at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
	at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6887)
	at clojure.core$reduce.invoke(core.clj:6869)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213.invoke(threaded.clj:21)
	at clojure.core$comp$fn__5876.invoke(core.clj:2587)
	at methodical.impl.combo.threaded$combine_with_threader$fn__18223.invoke(threaded.clj:43)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at clojure.core$partial$fn__5908.invoke(core.clj:2642)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
	at toucan2.connection$do_with_connection_primary_method_.invokeStatic(connection.clj:204)
	at toucan2.connection$do_with_connection_primary_method_.invoke(connection.clj:194)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at clojure.core$partial$fn__5908.invoke(core.clj:2642)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at methodical.impl.combo.threaded$fn__18243$fn__18244$fn__18245.invoke(threaded.clj:70)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213$fn__18217.invoke(threaded.clj:23)
	at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
	at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
	at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6887)
	at clojure.core$reduce.invoke(core.clj:6869)
	at methodical.impl.combo.threaded$reducer_fn$fn__18213.invoke(threaded.clj:21)
	at clojure.core$comp$fn__5876.invoke(core.clj:2587)
	at methodical.impl.combo.threaded$combine_with_threader$fn__18223.invoke(threaded.clj:43)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
	at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:436)
	at clojure.core$partial$fn__5908.invoke(core.clj:2642)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
	at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
	at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
	at metabase.db.custom_migrations.DeleteScanFieldValuesTriggerForDBThatTurnItOff.execute(custom_migrations.clj:1106)
	at liquibase.change.custom.CustomChangeWrapper.generateStatements(CustomChangeWrapper.java:169)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1271)
	at metabase.db.liquibase.h2.proxy$liquibase.database.core.H2Database$ff19274a.executeStatements(Unknown Source)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:755)
	... 72 more
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "General error: ""org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]"" [50000-214]"; SQL statement:
COMMIT [50000-214]
	... 226 more
Caused by: org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.write(MVStore.java:1271)
	at org.h2.mvstore.MVStore.storeBuffer(MVStore.java:1750)
	at org.h2.mvstore.MVStore.lambda$serializeAndStore$5(MVStore.java:1600)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 13 more
Caused by: org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@4b737e1c failed; length 122880 at 6115328 [2.1.214/2]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:493)
	at org.h2.mvstore.FileStore.writeFully(FileStore.java:113)
	at org.h2.mvstore.MVStore.write(MVStore.java:1269)
	... 7 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.FileChannelImpl.ensureOpen(Unknown Source)
	at java.base/sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:489)
	... 9 more
2024-06-04 04:55:49,888 INFO metabase.core :: Metabase Shutting Down ...
2024-06-04 04:55:49,889 INFO metabase.server :: Shutting Down Embedded Jetty Webserver
2024-06-04 04:55:49,904 WARN db.liquibase :: ()
2024-06-04 04:55:49,905 INFO metabase.core :: Metabase Shutdown COMPLETE
```

**Comment by paoliniluis** [comment]
can you tell us your pc/laptop specs?

**Comment by mikich** [comment]
I have the same issue - actually roadblock! Please, any suggestions?!?

Running the latest Docker 4.30.0 (149282) on Apple M3 Pro, 36Gb RAM, Sonoma 14.5


```
2024-06-04 08:37:19 Warning: environ value jdk-11.0.23+9 for key :java-version has been overwritten with 11.0.23
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # A fatal error has been detected by the Java Runtime Environment:
2024-06-04 08:37:20 #
2024-06-04 08:37:20 #  SIGSEGV (0xb) at pc=0x00007ffffebd9f3d, pid=1, tid=41
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
2024-06-04 08:37:20 # Java VM: OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (11.0.23+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
2024-06-04 08:37:20 # Problematic frame:
2024-06-04 08:37:20 # V  [libjvm.so+0x78cf3d]  G1ParEvacuateFollowersClosure::do_void()+0x1fd
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # Can not save log file, dump to screen..
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # A fatal error has been detected by the Java Runtime Environment:
2024-06-04 08:37:20 #
2024-06-04 08:37:20 #  SIGSEGV (0xb) at pc=0x00007ffffebd9f3d, pid=1, tid=41
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
2024-06-04 08:37:20 # Java VM: OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (11.0.23+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
2024-06-04 08:37:20 # Problematic frame:
2024-06-04 08:37:20 # V  [libjvm.so+0x78cf3d]  G1ParEvacuateFollowersClosure::do_void()+0x1fd
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-06-04 08:37:20 #
2024-06-04 08:37:20 # If you would like to submit a bug report, please visit:
2024-06-04 08:37:20 #   https://github.com/adoptium/adoptium-support/issues
2024-06-04 08:37:20 #
2024-06-04 08:37:20 
2024-06-04 08:37:20 ---------------  S U M M A R Y ------------
2024-06-04 08:37:20 
2024-06-04 08:37:20 Command Line: -XX:+IgnoreUnrecognizedVMOptions -Dfile.encoding=UTF-8 -Dlogfile.path=target/log -XX:+CrashOnOutOfMemoryError /app/metabase.jar
2024-06-04 08:37:20 
2024-06-04 08:37:20 Host: x86_64, 11 cores, 21G, Alpine Linux v3.19
2024-06-04 08:37:20 Time: Tue Jun  4 13:37:20 2024 UTC elapsed time: 4.173301 seconds (0d 0h 0m 4s)
2024-06-04 08:37:20 
2024-06-04 08:37:20 ---------------  T H R E A D  ---------------
Bla, Bla, Bla, Bla, Bla, Bla, Bla, ...
```

**Comment by markbastian** [comment]
Seeing the same thing. I've tried both latest and v0.49.13.

```
Hardware Overview:

  Model Name:	MacBook Pro
  Model Identifier:	Mac15,9
  Model Number:	Z1AH000VNLL/A
  Chip:	Apple M3 Max
  Total Number of Cores:	16 (12 performance and 4 efficiency)
  Memory:	64 GB
  System Firmware Version:	10151.121.1
  OS Loader Version:	10151.121.1
```
Error:
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007ffffebde144[thread 91 also had an error]
[thread 89 also had an error]
[thread 92 also had an error]
[thread 96 also had an error]
[thread 24 also had an error]
[thread 90 also had an error]
[thread 46 also had an error]
[thread 95 also had an error]
[thread 94 also had an error]
[thread 66 also had an error]
[thread 60 also had an error]
[thread 93 also had an error]
, pid=1, tid=54
#
# JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
# Java VM: OpenJDK 64-Bit Server VM Temurin-11.0.23+9 (11.0.23+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# V  [libjvm.so+0x791144]  oopDesc::size()+0x24
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
```

**Comment by paoliniluis** [comment]
Anyone that runs on Mac please upgrade your docker runtime to the latest version, or run with Colima, or build your own container of Metabase based on Ubuntu or other base image that supports ARM

you’re all most probably seeing https://github.com/docker/for-mac/issues/7172

btw, hi @markbastian, we miss you ❤️

**Comment by markbastian** [comment]
Thanks for the tip @paoliniluis.

I can confirm that the following steps worked for me:
1. `brew install colima`
2. `colima start`
3. `docker run -p 3100:3000 --platform linux/amd64 --name metabase-latest metabase/metabase`

Note that you can prevent the need to restart colima on startup with `brew services start colima`

Miss you all, too!

**Comment by markbastian** [comment]
Note [additional step required to work with compose](https://github.com/abiosoft/colima/discussions/874).

**Comment by mikich** [comment]
I tried all the above tips, thank you @markbastian , although unsuccessful:
Both:
- colima start
- docker run -p 3100:3000 --platform linux/amd64 --name metabase-latest metabase/metabase
And:
- docker-compose -f docker-compose.yml up --detach
Got me the same result as before :-(

> # A fatal error has been detected by the Java Runtime Environment:
> #
> #  SIGSEGV (0xb) at pc=0x00007ffffebe162e, pid=1, tid=55
> #
> # JRE version: OpenJDK Runtime Environment Temurin-11.0.23+9 (11.0.23+9) (build 11.0.23+9)
>

**Comment by schlos** [comment]
1) With Docker Desktop (macOS):

```
docker run -d -p 3000:3000 \
  -v ~/metabase-data:/Users/username/Workspace/metabase.db \
  -e "MB_DB_FILE=/metabase-data/metabase.db" \
  --name metabase metabase/metabase
```

> WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
> 792584cecb86e8b5dc56dbd0e0f9ac3c7bfce2f8cdefedceb70081449b86a5f0

2) With Colima (macOS):

```
colima start
INFO[0000] starting colima
INFO[0000] runtime: docker
INFO[0001] starting ...                                  context=vm
INFO[0012] provisioning ...                              context=docker
INFO[0012] starting ...                                  context=docker
INFO[0013] done
```

```
docker run -d -p 3000:3000 \
  -v ~/metabase-data:/Users/username/Workspace/metabase.db \
  -e "MB_DB_FILE=/metabase-data/metabase.db" \
  --name metabase metabase/metabase

Unable to find image 'metabase/metabase:latest' locally
latest: Pulling from metabase/metabase
4abcf2066143: Pull complete
a21a63612cbe: Pull complete
1e3d1cfa9847: Pull complete
c29f1605bed2: Pull complete
b17f30e766d9: Pull complete
09d6e4610ee2: Pull complete
85ebd572b17f: Pull complete
Digest: sha256:3531a3488e5238b537b76d23e267af961c88a15c6eb748ee6f829751d8fc867d
Status: Downloaded newer image for metabase/metabase:latest
```

> WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
> 9b8ec80003235e9c6a4887f98abc75b045a7de05a6aebcb2035524006472f4af

Am I doing something wrong?

**Comment by schlos** [comment]
> can you tell us your pc/laptop specs?

@paoliniluis hope this helps:

```
$ system_profiler SPHardwareDataType

Hardware:

    Hardware Overview:

      Model Name: MacBook Air
      Model Identifier: Mac15,13
      Model Number: Z1BP000MHX/A
      Chip: Apple M3
      Total Number of Cores: 8 (4 performance and 4 efficiency)
      Memory: 16 GB
      System Firmware Version: 10151.121.1
      OS Loader Version: 10151.121.1
```

**Comment by renansigolo** [comment]
Just had the same issue and solved by using this 2 steps:
1. In your terminal run `export LD_BIND_NOW=1`
1. Disable the Rosetta emulation from docker (I'm currently running docker v4.31.0)


<img width="1804" alt="Screenshot 2024-06-25 at 10 09 28 AM" src="https://github.com/metabase/metabase/assets/25624875/b7304b51-726c-4fec-9a8e-08ebfde10130">

I hope this helps!

**Comment by AykutCevik** [comment]
@renansigolo Thanks for the tip! This helped me to run metabase without installing anything else.

**Comment by mikich** [comment]
I ended up using an alternative image, specially created for arm64 (I'm running it on Mac M3Pro).
Here is my `docker-config.yml`:

`version: "4.0"
services:
  # BI - Metabse
  metabase:
    container_name: metabase
    platform: linux/arm64
    expose:
      - 3003
    ports:
      - "3003:3000"
    image: bobblybook/metabase
    restart: always
    networks:
      - app_net
# ===========================================================================================================
networks:
  app_net:
    ipam:
      driver: default
      config:
        - subnet: "172.16.240.0/24"`

**Comment by paoliniluis** [comment]
duplicate of https://github.com/metabase/metabase/issues/13119
